samples/app-ReporterPlus/ReporterPlus/Helpers/BlobHelper.cs

```diff
@@ -0,0 +1,160 @@
+﻿using AdaptiveCards.Templating;
+using Microsoft.Bot.Builder;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Azure.Storage.Blobs;
+using ReporterPlus.Models;
+using Azure.Storage.Blobs.Models;
+
+namespace ReporterPlus.Helpers
+{
+    public class BlobHelper
+    {
+        public static async Task<string> UploadToBlob(TaskModuleSubmitDataDeserializer taskModuleOutput, ITurnContext turnContext)
+        {
+            List<string> imageURL = new List<string>();
+            string audioURL = taskModuleOutput.data.RecorderOutput;
+            string requestID = taskModuleOutput.data.RequestId;
+            byte[] streamConverter;
+            string assignedToUserImageUrl;
+            string submittedByUserImageUrl;
+            List<Images> imgURL = new List<Images> {  };
+            var blobHttpHeader = new BlobHttpHeaders();
+            blobHttpHeader.ContentType = "image/jpeg";
+
+            BlobContainerClient container = new BlobContainerClient(Constants.BlobConnectionString, Constants.BlobContainerName);
+            BlobClient blob;
+
+            var assignedToUserImage = GetImage(taskModuleOutput.data.AssignedTo.objectId).Result;
+            if(assignedToUserImage != null)
+            {
+                blob = container.GetBlobClient(requestID + "-" + taskModuleOutput.data.AssignedTo.objectId + ".jpg");
+                var data = blob.UploadAsync(assignedToUserImage, blobHttpHeader);
```

Comment:
I don't think we are using the `data` variable anywhere. Maybe we can just call the `blob.UploadAsync(assignedToUserImage, blobHttpHeader);` method. Same in line number 51